pymysql/cursors.py

```diff
@@ -93,14 +94,32 @@ def _nextset(self, unbuffered=False):
     def nextset(self):
         return self._nextset(False)
 
+    def _ensure_bytes(self, x, encoding=None):
+        if not PY2:
+            return x
```

Comment:
Is this `if not PY2` required?
This behavior is not predictable from method name.